### ISSUE TITLE
edit a field's name

### ISSUE THREAD

**Issue by amnonkhen** [ebi-ait/checklist#86]
The name is the field's identifier. The label is the human readable name.
The field's ID is made of the field's name and version concatenated and separated by a colon, e.g. `collection_date:1.2`.
A change to a field's label means it is also assigned a new name, and that a whole new field record is created.

**Comment by theisuru** [comment]
if name and ID changes, does it become a whole new field and versions starts from 1.0?
Does this mean we do not update checklists that were using the old field?

**Comment by Jeena-Rajan** [comment]
Clarify with Peter and Colman

**Comment by Wkt8** [comment]
From Peter (in the checklist-facelift channel):

> Yes in my head, an updated field_name would make the previous checklist incompatible so would be a new version. Field name changes are rare in between GSC MIxS version releases(every 2-3 years). A consideration is that when there are changes e.g. due to a new GSC MIxS version, there could be multiple field name changes; ideally a version update of the checklists would handle say the 3 field_name changes in the same checklist as just one version. If this is going to be a pain in the posterior, this would be something to add to our mutual next verbal meet-up.

Tl;dr when a field name updates, the field version updates, and all checklists using that field switch to the latest version with the new field name. This is the behaviour as expected.

**Comment by Wkt8** [comment]
Is this done?

**Comment by Jeena-Rajan** [comment]
Linked to #185
